src/network/objectracker.py

```diff
@@ -42,21 +47,24 @@ def __init__(self):
         self.lastCleaned = time.time()
 
     def initInvBloom(self):
+        """"""
         if haveBloom:
             # lock?
             self.invBloom = BloomFilter(capacity=ObjectTracker.invInitialCapacity,
                                         error_rate=ObjectTracker.invErrorRate)
 
     def initAddrBloom(self):
+        """"""
```

Comment:
Init bloom filter for tracking addrs, WIP. This either needs to be moved to `addrthread.py` or removed.